addons/sale_project/models/sale_order.py

```diff
@@ -246,7 +248,7 @@ def _timesheet_create_task_prepare_values(self, project):
         title = sale_line_name_parts[0] or self.product_id.name
         description = '<br/>'.join(sale_line_name_parts[1:])
         return {
-            'name': title if project.sale_line_id else '%s: %s' % (self.order_id.name or '', title),
+            'name': title if project.sale_line_id else '%s- %s' % (self.order_id.name or '', title),
```

Comment:
Is it intended we don't have a space before `-`? 🤔

Reply:
Done. Thanks :slightly_smiling_face: